{
  "errors": [
    {
      "code": 2,
      "level": "error",
      "message": "Error while matching: Semgrep encountered an internal error.\nsemgrep-core exit code: 0\nsemgrep-core command: /home/runner/work/semgrep/semgrep/semgrep/semgrep/bin/semgrep-core -json -rules /tmp/tmpe0dix70a.yaml -j 2 -targets /tmp/tmpnb0q8ow0 -timeout 30 -max_memory 0 -json_time -use_parsing_cache /tmp/tmp3tm0ky1j -fast\nunexpected non-json output while invoking semgrep-core:\n--- semgrep-core stdout ---\ndone{\"matches\":[],\"errors\":[],\"skipped\":[],\"stats\":{\"okfiles\":0,\"errorfiles\":0},\"time\":{\"targets\":[],\"rules\":[\"rules.assert-eqeq-is-ok\",\"rules.eqeq-is-bad\",\"rules.python37-compatability-os-module\",\"rules.javascript-basic-eqeq-bad\"],\"rules_parse_time\":0.007333040237426758}}\n--- end semgrep-core stdout ---\n--- semgrep-core stderr ---\nExisting stack limits: Soft: 16777216, Hard: -1\nTrying to set soft limit to 16777216\nSet stack limit to 16777216, -1\n--- end semgrep-core stderr ---\nAn error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep",
      "type": "SemgrepError"
    }
  ],
  "results": []
}